### ISSUE TITLE
DNS poisoning

### ISSUE THREAD

**Issue by ecthros** [ecthros/pina-colada#9]
https://github.com/henryw14/DNS-Cache-Poisoning-with-Scapy/tree/master/code

**Comment by ecthros** [comment]
Not working right now - I can't figure out how to forward the traffic

**Comment by Kkevsterrr** [comment]
Did you set the kernel to forward traffic? Check wireshark to see what's going on

**Comment by ecthros** [comment]
I also tried to use kali's dnsspoof tool and that didn't work either :(

**Comment by Kkevsterrr** [comment]
Yeah if dns poisoning isn't even working in kali's build, I would just name it a dnsdos and call it a day :)

**Comment by ecthros** [comment]
This might also be because of the router im using - ill be working on another one in a week or so so I can try it there
